pkg/services/ngalert/notifier/available_channels.go

```diff
@@ -637,6 +637,28 @@ func GetAvailableNotifiers() []*alerting.NotifierPlugin {
 				},
 			},
 		},
+		{
+			Type:        "wecom",
+			Name:        "WeCom",
+			Description: "Sends HTTP POST request to WeCom",
+			Heading:     "WeCom settings",
+			Options: []alerting.NotifierOption{
+				{
+					Label:        "Url",
+					Element:      alerting.ElementTypeInput,
+					InputType:    alerting.InputTypeText,
+					Placeholder:  "https://qyapi.weixin.qq.com/cgi-bin/webhook/send?key=xxxxxxxx",
+					PropertyName: "url",
+					Required:     true,
+				},
+				{ // New in 8.0.
+					Label:        "Message",
+					Description:  "Custom wecom message. You can use template variables",
+					Element:      alerting.ElementTypeTextArea,
+					PropertyName: "message",
+				},
+			},
+		},
```

Comment:
How come we allow two types in legacy but one in ngalert - we should have the same on both. For simplicity, can we only have markdown on both of them?

If we are, feel free to ignore my comments about the allowedMsgTypes. But I would like to understand it first.

Reply:
You are right. If unifed alerting doesn't support image anymore, it'd be better to remove WeCom news type